recipes/dbcppp/all/patches/0001-dep-cmake-rem-kcd.patch

```diff
@@ -0,0 +1,316 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d905b29..ea94456 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,53 +5,17 @@ project("libdbcppp" VERSION 0.1.0)
+ 
+ option(build_tests "Build tests" ON)
+ option(build_examples "Build examples" ON)
++option(build_tools "Build examples" ON)
```

Comment:
Could you please open a PR to the upstream providing this PR?

```suggestion
+option(build_tools "Build tools" ON)
```

Reply:
Don't quite understand. do you mean use the patch for a PR in upstream?

Reply:
Yes! When ever we see contributions that would benefit everyone we like to encourage sharing them